packages/fast-components-react-msft/app/app.tsx

```diff
@@ -165,6 +185,12 @@ export default class App extends React.Component<{}, IAppState> {
         );
     }
 
+    private handleDensityUpdate = (e: React.ChangeEvent<HTMLInputElement>): void => {
+        const value: number = parseInt(e.target.value, 10);
+        this.setState({
+            density: value
```

Comment:
you can remove adding the const and just put `density: parseInt(e.target.value)`

Reply:
updated